src/V3EmitMkJson.cpp

```diff
@@ -0,0 +1,320 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Emit JSON manifest file
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2004-2025 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "V3PchAstNoMT.h"  // VL_MT_DISABLED_CODE_UNIT
+
+#include "V3EmitMkJson.h"
+
+#include "V3EmitCBase.h"
+#include "V3HierBlock.h"
+#include "V3Os.h"
+
+#include <memory>
+
+VL_DEFINE_DEBUG_FUNCTIONS;
+
+// ######################################################################
+//  Emit statements
+
+class V3EmitMkJsonEmitter final {
+    class Printer final {
+        // MEMBERS
+    private:
+        const std::unique_ptr<std::ofstream>& m_of;
+        std::stack<char> m_scope;  // stack of ']' and '}' to be used to close currently open scopes
```

Comment:
```suggestion
        std::stack<char> m_scope;  // Stack of ']' and '}' to be used to close currently open scopes
```

Capitalize comments please.  Many others.